app/client/components/productPage.tsx

```diff
@@ -418,14 +418,15 @@ const getProductDetailRenderer = (
             )}
             {productType.delivery?.showAddress?.(subscription) && (
               <ProductDetailRow
-                label="Delivery address"
+                label="Delivery address and instructions"
```

Comment:
If we are have this as the row label on the subs list, do we not need to display the instructions there too (for Home Delivery)?

Reply:
as discussed we'll possibly do this in follow-up PR